src/app/GitCommands/RevisionReader.cs

```diff
@@ -38,9 +38,9 @@ public sealed class RevisionReader
         /* Notes placeholder */ "{1}";
 
     private const string _reflogSelectorFormat = "%gD%n";
-    private const string _notesPrefix = "Notes:";
-    private const string _notesMarker = $"\n{_notesPrefix}";
-    private const string _notesFormat = $"%n{_notesPrefix}%n%N";
+    internal const string NotesPrefix = "\u0578\u043et\u0435\u0282:"; // Unicode l00k-alikes, ոоtеʂ:
```

Comment:
Maybe, I would prefer to have an uppercase char
https://www.compart.com/en/unicode/U+0548
```suggestion
    internal const string NotesPrefix = "\u0548\u043et\u0435\u0282:"; // Unicode l00k-alikes, ոоtеʂ:
```